HW1/P3/P3.py

```diff
@@ -0,0 +1,27 @@
+# anagram list maker
+
+# initiallizing spark
+from pyspark import SparkContext, SparkConf
+sc = SparkContext(conf=SparkConf())
+sc.setLogLevel("ERROR")
+
+# get word list from file
+# NOTE: copied text from given file into 'EOWL_words.txt'
+wlist = sc.textFile('HW1/P3/EOWL_words.txt',use_unicode=False)
+
+# make parallelized word list
+words = sc.parallelize(wlist.collect())
```

Comment:
No need to call parallelize after textFile. You can basically just use wlist as your basic rdd.